bucket_AB/ART/specification

```diff
@@ -0,0 +1,43 @@
+DEF[PORTVERSION]=	1.24.4
+# ----------------------------------------------------------------------------
+
+NAMEBASE=		ART
+VERSION=		${PORTVERSION}
+KEYWORDS=		graphics
+VARIANTS=		std
+SDESC[std]=		Raw image processing program
+HOMEPAGE=		https://art.pixls.us/
+CONTACT=		Michael_Neumann[mneumann@ntecs.de]
+
+DOWNLOAD_GROUPS=	main
+SITES[main]=		GITHUB/artpixls:ART:${PORTVERSION}
+DISTFILE[1]=		generated:main
+
+SPKGS[std]=		primary
+
+OPTIONS_AVAILABLE=	none
+OPTIONS_STANDARD=	none
+
+USES=			cmake c++:primary jpeg tiff pkgconfig
+
+# TODO: OpenColorIO
+CMAKE_ARGS=		-DENABLE_OCIO:BOOL=OFF
+
+BUILD_DEPENDS=		libcanberra:dev:std
+			fftw:single:float
+			libraw:dev:std
+			exiv2:dev:std
+
+BUILDRUN_DEPENDS=	exiv2:primary:std
```

Comment:
again, not wrong, but conventionally we put BUILD_DEPENDS, RUN_DEPENDS, and BUILDRUN_DEPENDS after OPTIONS and LICENSE block and before USES definition